app/models/spotlight/attachment.rb

```diff
@@ -4,11 +4,18 @@ module Spotlight
   ##
   # Sir-trevor image upload attachments
   class Attachment < ActiveRecord::Base
+    # Open to alternatives on how to do this, but this works
+    include Rails.application.routes.url_helpers
+
     belongs_to :exhibit
-    mount_uploader :file, Spotlight::AttachmentUploader
+    has_one_attached :file
 
-    def as_json(options = nil)
-      file.as_json(options).merge(name: name, uid: uid, attachment: to_global_id)
+    def as_json(_options = nil)
```

Comment:
This is super weird, but seems like we need to provide the accessible url at this point for our uploader forms to work correctly.